server/activate.go

```diff
@@ -182,7 +182,7 @@ func (p *Plugin) OnActivate() error {
 		rtcServerConfig.TURNConfig.StaticAuthSecret = cfg.TURNStaticAuthSecret
 	}
 	if cfg.ICEHostPortOverride != nil {
-		rtcServerConfig.ICEHostPortOverride = *cfg.ICEHostPortOverride
+		rtcServerConfig.ICEHostPortOverride = rtc.ICEHostPortOverride(fmt.Sprintf("%d", *cfg.ICEHostPortOverride))
```

Comment:
I think it's reasonable to keep supporting the basic syntax (single port) only from this side. We can always review later if needed but for anything advanced I'd say they should be using `rtcd`.